src/Schema/Directives/WhereDirective.php

```diff
@@ -52,12 +57,27 @@ public static function definition(): string
 
     public function handleBuilder(QueryBuilder|EloquentBuilder|Relation $builder, $value): QueryBuilder|EloquentBuilder|Relation
     {
+        if ($value === null && $this->directiveArgValue('ignoreNull', false)) {
+            return $builder;
+        }
+
+        $operator = $this->directiveArgValue('operator', '=');
+
+        // If the client sends an illegal value to the used operator it would crash the query
```

Comment:
Just doing nothing seems very unexpected and hard to debug.

This change also seems unnecessary for what this pull request is trying to do. Conflating distinct changes in a single pull request slows the review process - I would merge now if this change were not added on top.

Reply:
Sure I'll remove it from this PR but I think that the illegal operator and value combination exception should be catched instead of throwing it to the client, how should it be done?

Reply:
I don't think so, errors should definitely be surfaced to the client. Swallowing the error and defaulting to a no-op is just about the worst possible course of action. A possible improvement would be to provide a better error message.